### DISCUSSION TITLE
Работа в здравоохранении всем!

### DISCUSSION THREAD

**Discussion by rasusibtisam** [octothorp-team/public-dobrograd#622] [Предложения]
**Предисловие:**
После появления функциональной профессии врача на ДБГ, на которую можно попасть только через WhiteList, я смог участвовать в достаточно многих ситуациях, как врач, связанных с оказанием мед. помощи. Функция "вызова врача" по телефону наконец приобрела смысл. Игроки что делали вызов рача наконец встречали кого-то похожего на врача, а не офицера-мастера на все руки. Много вызовов на перестрелки, хир. операции проведённые РП путём - это показало что есть достаточно большой спрос на услуги врача. Игра за полицию требовала вызывать карету скорой помощи для пострадавших, а не неловкую отыгровку "/it приехали врачи". Больница после своего появления стала ощущаться для меня как неотъемлемая часть профессий сервера, как профа копы. Однако вайт лист на врача не имел большой спрос из-за своего порога вхождения.

После закрытия фракции и отсутствия медиков, дальнейшие отыгровки по оказанию помощи и доставке пострадавших в больницу в тесных полицейских бобиках стали выглядеть крайне постыдно в плане отыгровки, это стало нелогично, особенно когда игроки осознают, что есть профессия врача и почва для отыгрыша больницы-рп. Вся эта ситуация показала мне, что больница, как "фракция" предоставляет необходимые для citizen rp услуги, особенно если вспомнить, что на ДБГ перестрелки, драки и иного рода насилие являются не редкостью. 

В моём видении профессия работника больницы также необходима, как  и профессия копа, где первый лечит пострадавших после перестрелки, а второй защищает порядок и нередко применяет насилие, чтобы угомонить взбунтовавшихся краймеров и вот моё 

 **Предложение:**

• Сделать профессию врача(парамедика) в больнице общедоступной,: все игроки смогут выбрать профу врача, также как и профу копа, токаря и т.д.
Преимущества:
1) плюсик к развитию civil rp
2) Функция "вызов скорой помощи" будет вызывать врачей, а не полицию. Также приобретет правильный смысл в глазах игрока, ведь где это видано чтобы полицейский ехал на вызовы скорой помощи?
3) разнообразие геймплея - у игроков появится новая занятость не связанная с краймом или полицией, где они смогут развеется, а не участвовать в вооржуенном конфликте
4) Восстановление логики игрового мира - "фракция" больницы появилась и успела сделать немало отыгровок, после которых я стал испытывать нужду в ней, особенно когда дело касается пострадавших в перестрелках, ведь правильнее отвезти в больницу на отыгровку выздоровления, чем просто помахать механ аптечкой перед игроком.

upd Желательно чтобы была профу назвали "врач", чтоб в одной профе сочетались все услуги разных врачей, который может она предоставить исходя из воображения и навыков игроков, а также "главврач", поскольку это гос. учреждение, то им кто-то должен заведовать и избавлять от некомпитетных работников

**Comment by Kyper1** [comment]
Через WhiteList, я так полагаю, сделано для того, чтобы игроки, не разбирающиеся в медицине, не бегали за этой профессией. Как ты и сказал "сам геймплей, по идее, это эмоуты и использование механ аптечки после эмоутов" и ты наверняка замечаешь разницу между парамедиком и полицейским медиком, в плане отыгровок, а замечаешь ты ее потому, что парамедик не общедоступная профессия. Я клоню к тому, что если сделать эту профессию доступной для всех, то, вышеупомянутые мною, игроки начнут разбирать все слоты и нормальные парамедики просто будут никому не нужны.

**Reply by rasusibtisam** [reply]
Отыгровки эмоутов для лечения делают все копы, в их аптечке, по логике вещей, нет всех медикаментов на земле, тем самым невозможно вылечить перелом, достать пулю из тела и т.п. вещи которые могут сделать специалисты из больницы.

Сама профессия полицейского тоже требуется знания для нормального обеспечения правопорядка, но всё-таки как игровая условность для semi-serious рп сервера работа в полиции является общедоступной. Так повелось с самого зарождения Дарк рп серверов. 

Профессия работника больницы тоже является значимой для ДБГ, я это осознал после игры на Платтсбурге, а также увидев работу парамедиков в действии на ДБГ. Наличие здания больницы на ДБГ, готовая профессия парамедиков, механически реализованная карета скорой помощи, высокий спрос на услуги лечения, но неэффективность вайт-листа в плане набора людей, а также давно сделанная функция "вызова скорой помощи", говорит о том, что работа в больнице должна стать доступной всем игрокам говорит само за себя. Смысл ограничивать доступ к данной профессии просто неблагоприятно скажется на логике, ведь если есть всё готовое, а фракция сама по себе закрыта(заморожена, но кто знает когда они захотят), то не лучше ли превратить работу в больнице в профессию доступную для всех игроков?

 Мы как игроки выбирая профу за которую играем не должны получать образование в соответствующей сфере. Игра за повара требует знаний тоже; за токаря - аналогично,;  за священнослужителя - красноречие, проповеди; за копа - знания УПК, УК, ДК и т.д. 

Больница это гос. Учреждение, как ПД, которое никому не принадлежит, т.к. сделана профа ПД не игроком, а создателем сервера, что должно быть и с больницей. Всё необходимое для начала отыгровки за медика имеется: здание, транспорт, модели; сама работа в больнице на большинстве серверов Дарк рп является доступной всем, а также чуть ли не часов первой доступной профоц на большинстве серверах. Не лучше ли дать игрокам возможность реализовать себя в гос. Структуре не связанной с насилием и конфликтами, а со спасением жизней?
